Core/Core/Inbox/AttachmentPicker/ViewModel/AttachmentPickerViewModel.swift

```diff
@@ -165,5 +211,27 @@ class AttachmentPickerViewModel: ObservableObject {
                 self?.interactor.removeFile(file: file)
             }
             .store(in: &subscriptions)
+
+        deleteFileButtonDidTap
+            .flatMap { [weak self] file in
+                if let self {
+                    return self.interactor.deleteFile(file: file)
+                } else {
+                    return Just(()).eraseToAnyPublisher()
+                }
+            }
+            .sink()
+            .store(in: &subscriptions)
+
+        fileSelected.sink(receiveCompletion: { _ in }, receiveValue: { [weak self] (controller, file) in
```

Comment:
This could be simplified to `sink(receiveValue: {` or `sink {`, unless I'm mistaken